### PR TITLE
Update a CHANGELOG.md for v0.8.0-beta

### PR DESCRIPTION
**Pull request by mykhailo-kuchma**
Relates-To: OLPEDGE-840

Signed-off-by: Mykhailo Kuchma <ext-mykhailo.kuchma@here.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,41 @@
...
+
+* Project renamed to `here-olp-sdk-cpp`
+* Project structure was changed, functional, integration, performance tests introduced.
+* Local OLP server that mimics OLP added.
+* `OLP_SDK_DEPRECATED` macro is added to deprecate API
+* **Breaking Change** `SendOutcome::IsSuccessful` typo fixed.
+* Missing dependencies for the iOS build as a shared library added.
+* `KeyValueCache` instance was added to `OlpClientSetting`.
+* Boost download source was changed.
+* Added a `Condition` class, a helper class allows waiting until the notification.
```
Comment (andescu) [inline CHANGELOG.md]:
```suggestion
* Added `Condition` class helping repositories sync wait on the OLP response.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,41 @@
...
+
+**olp-cpp-sdk-authentication**
+
+* **Breaking Change** `olp::authentication::Settings` are now require `olp::authentication::AuthenticationCredentials`.
+
+**olp-cpp-sdk-dataservice-read**
+
+* Added a new class VersionedLayerClient that is used to access versioned layers in OLP, class implements `GetData`, `GetPartitions`, `PrefetchTiles` methods from `CatalogClient`.
```
Comment (andescu) [inline CHANGELOG.md]:
```suggestion
* Added new class `VersionedLayerClient` that is used to access versioned layers in OLP. Class implements `GetData`, `GetPartitions`, `PrefetchTiles` methods from `CatalogClient`.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,41 @@
...
+* Added a new class VolatileLayerClient that is used to access volatile layers in OLP. class implements `GetPartitions`, `GetData` methods from `CatalogClient`.
```
Comment (andescu) [inline CHANGELOG.md]:
```suggestion
* Added new class `VolatileLayerClient` that is used to access volatile layers in OLP. Class implements `GetPartitions`, `GetData` methods from `CatalogClient`.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,41 @@
...
+* `GetData` method was reworked.
```
Comment (andescu) [inline CHANGELOG.md]: Remove

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,41 @@
...
+* `GetData`, `GetPartitions`, `PrefetchTiles` methods in `CatalogClient` deprecated
+* **Breaking Change** CatalogClient constructor changed, a cache passed with `OlpClientSetting`, settings passed by value.
```
Comment (andescu) [inline CHANGELOG.md]:
```suggestion
* **Breaking Change** CatalogClient constructor changed. `OlpClientSettings` must be passed by value, `KeyValueCache` is now part of `OlpClientSettings`.
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,41 @@
...
+* Added  `Types.h`, the header file contains all responses and callbacks types.
```
Comment (andescu) [inline CHANGELOG.md]:
```suggestion
* Moved all responses and callbacks aliases to  `Types.h`.
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,41 @@
...
+
+**olp-cpp-sdk-dataservice-write**
+
+* `StreamLayerClientSettings` class introduced.
+* **Breaking Change** `StreamLayerClient` costructor changed.
```
Comment (andescu) [inline CHANGELOG.md]:
```suggestion
* **Breaking Change** `StreamLayerClient` constructor changed.
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,41 @@
...
+* **Breaking Change** `model::FlushRequest` class was introduced. `StreamLayerClient::Flush` changed.
+* `FlushEventListener` removed.
```
Comment (andescu) [inline CHANGELOG.md]:
```suggestion
**Breaking Change** `StreamLayerClient::Flush` changed. Now taking `FlushRequest` as input and a `FlushCallback` to be triggered after the flush.
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,39 @@
...
+* `OLP_SDK_DEPRECATED` macro is added to deprecate API
+* **Breaking Change** `SendOutcome::IsSuccessful` typo fixed.
+* Missing dependencies for the iOS build as a shared library added.
+* `KeyValueCache` instance was added to `OlpClientSetting`.
+* Boost download source was changed.
+* Added `Condition` class helping repositories sync wait on the OLP response.
+
+**olp-cpp-sdk-authentication**
+
+* **Breaking Change** `olp::authentication::Settings` now require `olp::authentication::AuthenticationCredentials`.
```
Comment (leonid-ziskel) [inline CHANGELOG.md]: requires

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,3 +1,39 @@
+## v0.8.0-beta (31/10/2019)
+
+**Common**
+
+* Project renamed to `here-olp-sdk-cpp`
+* Project structure was changed, functional, integration, performance tests introduced.
+* Local OLP server that mimics OLP added.
+* `OLP_SDK_DEPRECATED` macro is added to deprecate API
```
Comment (leonid-ziskel) [inline CHANGELOG.md]: please append "." to align the the other points

**Pull request merged**